packages/ifpack2/src/Ifpack2_BlockRelaxation_def.hpp

```diff
@@ -171,6 +171,7 @@ getValidParameters () const
                                    typename MatrixType::node_type> > dummy;
   validParams->set("partitioner: coordinates",dummy);
   validParams->set("timer for apply", true);
+  validParams->set("Number of subparts per part", 1);
```

Comment:
Could this be named "partitioner: subparts per part"? That way it's easier to see where it's used when looking at an Ifpack2 parameter list